frontend/mr-admin-flate/src/mocks/fixtures/mock_avtaler.ts

```diff
@@ -21,7 +21,7 @@ export const mockAvtaler: PaginertAvtale = {
       },
       navn: "Testtiltak Varig",
       ansvarlig: {
-        navident: "B123456",
+        navident: "B99876",
```

Comment:
Hva er grunnen til å bytte ident her? Ser også at det er feil format, skal være en bokstav og 6 tall, ikke 5

Reply:
For å matche med mock-brukeren vår. Jeg kan ta en gjennomgang på den og se om vi kan få det konsekvent. Enig i at det blir litt uryddig sånn det er nå.